src/aat/resources/features/F-131 - AC CRUD external createEvent/S-131.5_CreateCase_Token_Creation.td.json

```diff
@@ -14,7 +14,8 @@
 
   "users": {
     "invokingUser": {
-      "_extends_": "BeftaCaseworker1"
+      "username": "master.caseworker@gmail.com",
```

Comment:
Could you update these instances of master.caseworker@gmail.com to be a defined user in `/common/users` like BeftaCaseworker1 was